pulseapi/entries/tests/test_member_views.py

```diff
@@ -148,3 +155,10 @@ def test_dual_tag_search(self):
         response_data = json.loads(str(response.content, 'utf-8'))
         count = response_data['count']
         self.assertEqual(count, 1)
+
+    def test_inactive_profile_entry_is_hidden(self):
+        response = self.client.get('/api/pulse/entries/?search=inactive_profile_entry&format=json')
+        self.assertEqual(response.status_code, 200)
+        response_data = json.loads(str(response.content, 'utf-8'))
+        count = response_data['count']
+        self.assertEqual(count, 0)

```

Comment:
**Suggestion** Similar to the other point, when we test exclusion, I think it makes sense to include another object that needs to be present as a sanity check to make sure our tests are actually working.

Reply:
Updated!